packages/zapp/console/src/components/Entities/strings.ts

```diff
@@ -1,15 +1,38 @@
 import { createLocalizedString } from '@flyteconsole/locale';
 import { startCase } from 'lodash';
 
+const launchStrings = (typeString: string) => `Launch ${startCase(typeString)}`;
+const noDescription = (typeString: string) => `This ${typeString} has no description.`;
+const noSchedules = (typeString: string) => `This ${typeString} has no schedules.`;
+const allExecutionsChartTitle = (typeString: string) =>
+  `All Executions in the ${startCase(typeString)}`;
+const versionsTitle = (typeString: string) => `Recent ${startCase(typeString)} Versions`;
+
 const str = {
-  allExecutionsChartTitle: 'All Executions in the Workflow',
-  workflowVersionsTitle: 'Recent Workflow Versions',
   viewAll: 'View All',
   schedulesHeader: 'Schedules',
   collapseButton: (showAll: boolean) => (showAll ? 'Collapse' : 'Expand'),
-  launchStrings: (typeString: string) => `Launch ${startCase(typeString)}`,
-  noDescription: (typeString: string) => `This ${typeString} has no description.`,
-  noSchedules: (typeString: string) => `This ${typeString} has no schedules.`,
+  launchStrings_workflow: launchStrings('Workflow'),
```

Comment:
nit: in real string files ready for localization we would never have functions inside, so ideally `This Workflow has no schedules.` should be used here and below instead of `launchStrings('Workflow')`.
Mainly note for future, as we anyway would need tp go though all our files before we will be ready to add real localization support.

Reply:
ohhh. I see now. that makes sense will fix it